frontend/packages/console-app/src/components/user-preferences/theme/ThemeField.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { usePreferredTheme } from './usePreferredTheme';
+
+enum ThemeTypes {
+  light = 'Light',
+  dark = 'Dark',
```

Comment:
We need one for "System default" as well (and that should be the default value when not set).

Reply:
And I believe we need a translation for them as well